probe_scraper/glean_checks.py

```diff
@@ -126,3 +127,81 @@ def check_for_duplicate_metrics(repositories, metrics_by_repo, emails):
         }
 
     return found_duplicates
+
+
+EXPIRED_METRICS_EMAIL_TEMPLATE = """
+The following metrics in {repo_name} will expire in the next {expire_days} days or have already expired.
+
+{expired_metrics}
+
+What to do about this:
+
+1. If the metric is no longer needed, remove it from the `metrics.yaml` file.
+2. If the metric is still required, resubmit a data review [1] and extend its expiration date.
+
+If you have any problems, please ask for help on the #glean Slack channel. We'll give you a hand.
+
+What happens if you don't fix this:
+
+The metrics will automatically stop collecting data after their expiration date, and you will continue to get this e-mail as a daily reminder.
```

Comment:
Due to how we do date-expiry, is it worth mentioning that this is affects _builds_ after the expiration date?

Reply:
Very good point.